generate-grammars/generate-mozcpp.sh

```diff
@@ -12,7 +12,7 @@ TS_CPP_CRATE="tree-sitter-cpp"
 JSON_CRATE_FILENAME=".cargo_vcs_info.json"
 
 # Get the current tree-sitter-cpp crate version from the tree-sitter-mozcpp grammar
-TS_CPP_VERSION=`grep $TS_CPP_CRATE tree-sitter-mozcpp/Cargo.toml | cut -f2 -d "," | cut -f2 -d "=" | tr -d ' ' | tr -d } | tr -d \"`
+TS_CPP_VERSION=`grep -m 1 $TS_CPP_CRATE tree-sitter-mozcpp/Cargo.toml | cut -f2 -d "," | cut -f2 -d "=" | tr -d ' ' | tr -d } | tr -d \"`
```

Comment:
Why is `-m 1` needed?

Reply:
To get the first occurrence of `tree-sitter-cpp`, otherwise we get two occurrences because we have add a `cargo-udeps` ignore